patroni/postgresql/slots.py

```diff
@@ -260,16 +262,19 @@ def load_replication_slots(self) -> None:
         """
         if self._postgresql.major_version >= 90400 and self._schedule_load_slots:
             replication_slots: Dict[str, Dict[str, Any]] = {}
-            extra = ", catalog_xmin, pg_catalog.pg_wal_lsn_diff(confirmed_flush_lsn, '0/0')::bigint" \
+            pg_wal_lsn_diff = f'pg_catalog.pg_{self._postgresql.wal_name}_{self._postgresql.lsn_name}_diff'
```

Comment:
I wonder if you would consider running `black` on code and having a test that errors if black would make changes to edited files? It might cut down on trivial formatting comments by standardising formatting.

```suggestion
            pg_wal_lsn_diff = f"pg_catalog.pg_{self._postgresql.wal_name}_{self._postgresql.lsn_name}_diff"
```